shoes-core/lib/shoes/color/dsl_helpers.rb

```diff
@@ -81,7 +83,11 @@ def image_file?(arg)
       end
 
       def image_pattern(path)
+        path = absolute_file_path(path)
         Shoes::ImagePattern.new path if File.exist?(path)
+      rescue FileNotFoundError => e
+        Shoes.logger.error(e.message)
+        return nil
```

Comment:
I wonder why we only rescure here and in no other place? Because we normally catch stuff elsewhere? But why here? Seems interesting to me.